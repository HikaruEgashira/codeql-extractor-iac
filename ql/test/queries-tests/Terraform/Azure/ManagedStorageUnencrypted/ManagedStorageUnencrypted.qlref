security/Terraform/Azure/ManagedStorageUnencrypted.ql