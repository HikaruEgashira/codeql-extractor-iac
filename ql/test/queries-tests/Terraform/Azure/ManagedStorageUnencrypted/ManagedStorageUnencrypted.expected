| storage.hcl:6:1:13:1 | ManagedDisk acctestmd | Azure Storage is Unencrypted for 'acctestmd' |
| storage.hcl:23:3:25:3 | ManagedDiskEncryptionSettings | Azure Storage is Unencrypted for 'acctestmd' |
